src/index.html

```diff
@@ -11,6 +11,43 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      action="http://localhost:8080" 
+      method="post"
+      data-qa="big"
+      class="container"
+    >
+      <img 
+        src="images/Search.svg" 
+        alt="search"
+        class="image"
+      >
+      <input 
+        type="search" 
+        name="q"
```

Comment:
i asked to rename it

Reply:
sorry, my bad, forget about second form(